Library/Formula/sdedit.rb

```diff
@@ -0,0 +1,16 @@
+class Sdedit < Formula
+  desc "Tool for generating sequence diagrams very quickly."
+  homepage "http://sdedit.sourceforge.net"
+  url "https://downloads.sourceforge.net/projects/sdedit/files/sdedit/4.0/sdedit-4.01.jar"
+  sha256 "86f52193ceb3a478fd0cb356f631b6131bc649561fcc5effe73edfb3b12c33c9"
+  depends_on :java
+
+  def install
+    libexec.install "sdedit-#{version}.jar"
+    bin.write_jar_script libexec/"sdedit-#{version}.jar", "sdedit"
+  end
+
+  test do
+    system "#{bin}/sdedit", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!